nucypher/cli/characters/alice.py

```diff
@@ -119,15 +119,27 @@ def alice(click_config,
         if dev:
             raise click.BadArgumentUsage("Cannot create a persistent development character")
 
+        if not provider_uri and not federated_only:
+            raise click.BadOptionUsage(option_name='--provider',
+                                       message="--provider is required to create a new decentralized alice.")
+
         if not config_root:                         # Flag
             config_root = click_config.config_file  # Envvar
 
-        new_alice_config = AliceConfiguration.generate(password=get_password(confirm=True),
+        if not pay_with and not federated_only:
```

Comment:
@KPrasch Just a re-iteration of my comment about `pay_with` from #1146 : https://github.com/nucypher/nucypher/pull/1146#discussion_r308214961. Not sure if you saw it.

Reply:
@derekpierre - I responded to your comment there.

Reply:
:heavy_check_mark: